### DISCUSSION TITLE
How to do localized pathnames with next-intl?

### DISCUSSION THREAD

**Discussion by empz** [tolgee/tolgee-platform#2485] [Q&A] [answered]
I'm looking to do what's described here:
https://next-intl-docs.vercel.app/docs/routing#pathnames

But the Next.js App router example doesn't make use of next-intl's `defineRouting` method.

**Comment by stepan662** [comment, marked as answer]
Hey, you should be able to add the `defineRouting` as described in the docs.
If the question is how to store localized paths in tolgee platform, I'd put them into a separated namespace, export them into a static file and then import them one by one into the `routing.ts` (I think there is no need for Tolgee SDK). You should be able to automate this with `@tolgee/cli`.

**Reply by RicSala** [reply]
there is no "defineRouting" in the docs, or at least I haven't found a mention of it...

**Reply by stepan662** [reply]
It's referrring to next-intl docs: https://next-intl.dev/docs/routing#definerouting

**Reply by RicSala** [reply]
Then I am missing something..

This is the Tolgee repo, he asks you how to do that (in Tolgee I suppose), and you answer to use Next-intl?

I am sure I am missing something and would love to get what :)

**Reply by stepan662** [reply]
You can use tolgee together with next-intl, we have a page in docs about this: https://docs.tolgee.io/js-sdk/integrations/react/next/app-router-next-intl

Tolgee doesn't have a deep next.js integration like next-intl, so that's why it's useful to use next-intl.
